### ISSUE TITLE
java.lang.ArithmeticException: / by zero

### ISSUE THREAD

**Issue by alinco8** [FiguraMC/Figura#156]
Occurs when trying to open Figura menu only when the screen is maximized

## Error
```log
java.lang.ArithmeticException: / by zero
	at forge.org.figuramc.figura.gui.widgets.BackendMotdWidget.renderScrollBar(BackendMotdWidget.java:153) ~[figura-0.1.2+1.18.2.jar%23107!/:?] {re:classloading}
	at forge.org.figuramc.figura.gui.widgets.BackendMotdWidget.renderDecorations(BackendMotdWidget.java:175) ~[figura-0.1.2+1.18.2.jar%23107!/:?] {re:classloading}
	at forge.org.figuramc.figura.gui.widgets.BackendMotdWidget.m_6303_(BackendMotdWidget.java:106) ~[figura-0.1.2+1.18.2.jar%23107!/:?] {re:classloading}
	at net.minecraft.client.gui.components.AbstractWidget.m_6305_(AbstractWidget.java:64) ~[client-1.18.2-20220404.173914-srg.jar%23174!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:fancymenu.general.mixin.json:MixinWidget,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.screens.Screen.m_6305_(Screen.java:104) ~[client-1.18.2-20220404.173914-srg.jar%23174!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:computing_frames,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:balm.mixins.json:ScreenAccessor,pl:mixin:APP:konkrete.mixin.json:client.MixinScreen,pl:mixin:APP:patchouli_xplat.mixins.json:client.AccessorScreen,pl:mixin:APP:quark.mixins.json:client.ScreenMixin,pl:mixin:APP:forge-figura-common.mixins.json:gui.ScreenAccessor,pl:mixin:APP:forge-figura-common.mixins.json:gui.ScreenMixin,pl:mixin:APP:fancymenu.general.mixin.json:MixinScreen,pl:mixin:APP:mixins.essential.json:client.gui.GuiScreenAccessor,pl:mixin:APP:mixins.essential.json:client.gui.MixinGuiScreen,pl:mixin:A,pl:runtimedistcleaner:A}
	at forge.org.figuramc.figura.gui.screens.AbstractPanelScreen.m_6305_(AbstractPanelScreen.java:85) ~[figura-0.1.2+1.18.2.jar%23107!/:?] {re:mixin,re:classloading}
	at net.minecraftforge.client.ForgeHooksClient.drawScreenInternal(ForgeHooksClient.java:403) ~[forge-1.18.2-40.2.14-universal.jar%23179!/:?] {re:mixin,re:classloading}
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:396) ~[forge-1.18.2-40.2.14-universal.jar%23179!/:?] {re:mixin,re:classloading}
	at net.minecraft.client.renderer.GameRenderer.m_109093_(GameRenderer.java:890) ~[client-1.18.2-20220404.173914-srg.jar%23174!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:pehkui.mixins.json:client.compat115plus.compat1192minus.GameRendererMixin,pl:mixin:APP:embeddiumplus.mixin.json:darkness.MixinGameRenderer,pl:mixin:APP:darkness.client.json:MixinGameRenderer,pl:mixin:APP:railways-common.mixins.json:conductor_possession.MixinGameRenderer,pl:mixin:APP:mixins.oculus.json:GameRendererAccessor,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer,pl:mixin:APP:mixins.oculus.json:MixinModelViewBobbing,pl:mixin:APP:mixins.oculus.json:MixinTweakFarPlane,pl:mixin:APP:fastload.mixins.json:client.GameRendererMixin,pl:mixin:APP:pehkui.mixins.json:client.compat115plus.GameRendererMixin,pl:mixin:APP:tac.mixins.json:client.GameRendererMixin,pl:mixin:APP:sodium-extra.mixins.json:prevent_shaders.MixinGameRenderer,pl:mixin:APP:forge-figura-common.mixins.json:render.GameRendererAccessor,pl:mixin:APP:forge-figura-common.mixins.json:render.GameRendererMixin,pl:mixin:APP:mixins.essential.json:client.renderer.MixinEntityRenderer,pl:mixin:APP:mixins.essential.json:client.renderer.MixinEntityRenderer_Zoom,pl:mixin:APP:create.mixins.json:accessor.GameRendererAccessor,pl:mixin:APP:create.mixins.json:client.GameRendererMixin,pl:mixin:APP:mixins.oculus.json:MixinGameRenderer_NightVisionCompat,pl:mixin:APP:mixins.essential.json:events.Mixin_GuiDrawScreenEvent_Priority,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1046) ~[client-1.18.2-20220404.173914-srg.jar%23174!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:665) ~[client-1.18.2-20220404.173914-srg.jar%23174!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:205) ~[client-1.18.2-20220404.173914-srg.jar%23174!/:?] {re:classloading,re:mixin,pl:runtimedistcleaner:A,pl:mixin:A,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:31) ~[fmlloader-1.18.2-40.2.14.jar%2352!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149) [bootstraplauncher-1.0.0.jar:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:66) [ForgeWrapper-1.5.7-prism.jar:1.5.7-prism] {}
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:87) [NewLaunch.jar:?] {}
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:130) [NewLaunch.jar:?] {}
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) [NewLaunch.jar:?] {}
```

## Log
https://mclo.gs/nN9BAUZ

## Mod List:
- Fantasy's Furniture 
- Krypton Reforged 
- Fast IP Ping 
- Chat Heads 
- ModernFix 
- Public GUI Announcement 
- Better Animations Collection 
- CreativeCore 
- FancyMenu 
- Ferrite Core 
- Oculus 
- Cloth Config v4 API 
- Configured 
- Too Fast 
- Better Mods Button 
- Farmer's Delight 
- betterfpsdist mod 
- Pineapple Delight 
- Selene 
- End's Delight 
- FastFurnace 
- True Darkness 
- Keyboard Wizard 
- SuperMartijn642's Config Library 
- Stylish Effects 
- Effortless Building 
- Mysterious Mountain Lib 
- Moving Elevators 
- ProjectRed Illumination 
- Another Furniture 
- Xaero's Map - Waystones Compability 
- YDM's Weapon Master 
- Kotlin for Forge 
- Fast Suite 
- FastLeafDecay 
- Jade 
- Balm 
- LootBeams 
- Supplementaries 
- ProjectRed Exploration 
- Cultural Delights 
- Faster Ladder Climbing 
- Embeddium++ 
- Create: Steam 'n Rails 
- ProjectRed Fabrication 
- Not Enough Crashes 
- Ocean's Delight 
- FPS Reducer 
- ProjectRed Core 
- ProjectRed Integration 
- Create Deco 
- Drippy Loading Screen 
- Figura 
- Gravestone Mod 
- ProjectRed Expansion 
- Inspirations 
- Sound Physics Remastered 
- ToroHealth 
- Patchouli 
- Mantle 
- Jade Addons 
- Rubidium Extra 
- Timeless and Classics Guns 
- AutoRegLib 
- Xaero's Minimap 
- MrCrayfish's Furniture Mod 
- Just Enough Items 
- Memory Leak Fix 
- Etched 
- Caelus API 
- SuperMartijn642's Core Lib 
- Create Sabers 
- Sophisticated Backpacks 
- Sophisticated Core 
- Framework 
- Konkrete 
- Lightspeed 
- Saturn 
- Effective (Forge) 
- ESSENTIAL Mod 
- LazyDFU 
- EnhancedVisuals 
- Pollen 
- Decoration Delight 
- Pehkui 
- Miner's Delight 
- Fastload-Reforged 
- TexTrue's Embeddium Options 
- Player Animator 
- Eating Animation 
- Crabber's Delight 
- Iron's Spells 'n Spellbooks 
- Simple Planes 
- Epic Samurai 
- Gpu memory leak fix 
- Mouse Tweaks 
- Puzzles Lib 
- CBMultipart 
- Fast Workbench 
- Ambient Sounds 
- Placebo 
- ReAuth 
- Chefs Delight 
- Curios API 
- Ender's Delight 
- ProjectRed Transmission 
- GeckoLib 
- Macaw's Furniture 
- Radon 
- PlayerRevive 
- Create 
- Embeddium 
- Paraglider 
- Corn Delight 
- ParCool! 
- Nethers Delight 
- CodeChicken Lib 
- DamageTilt 
- Create Big Cannons 
- Quark 
- Waystones 
- Delightful 
- Catalogue

**Comment by GrandpaScout** [comment]
This appears to be a duplicate of #76

**Comment by UnlikePaladin** [comment]
this is indeed a duplicate, closing it as such, plus it's already fixed on the release candidates for 0.1.3
